pkg/reconciler/reconciler.go

```diff
@@ -57,6 +62,29 @@ func NewReconciler(client client.Client, anycastTracker *anycast.AnycastTracker)
 		reconciler.config = config
 	}
 
+	cfg, err := clientgorest.InClusterConfig()
+	if err != nil {
+		return nil, err
+	}
+
+	c, err := clientgo.NewForConfig(cfg)
```

Comment:
same as above, consider using the controller-runtime client

Reply:
Done.

Used reconciler.client for this one as it's already created.